client/store/cart.js

```diff
@@ -88,7 +89,7 @@ export const cart = () => dispatch => {
 const reducer = (state = initialState, action) => {
   switch (action.type) {
     case SHOW_CART:
-      return state.cart
+      return {...state.cart}
```

Comment:
{...state, cart: [...cart]

Reply:
I will make the change, but I'm not certain how this works. Looking to review this section to fix the cart.